website/docs/tutorials/guides/testing.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Testing (6 min)
+sidebar_label: Testing (10 min)
```

Comment:
I don't think anybody will be able to read this page in 10 min. Can we just remove this time hints?

Reply:
Fine.
Personally, I find time hints convenient. I copied this approach from here:
https://developers.google.com/tech-writing/one

Reply:
I am unopposed either to have these hints or not 🤷🏻